app/ActionTypes.js

```diff
@@ -82,6 +82,9 @@ export const SAVE_HISTORY_VIDEO = 'SAVE_HISTORY_VIDEO';
 export const SAVE_END_SET_TIMER = 'SAVE_END_SET_TIMER';
 export const PRESENT_END_SET_TIMER = 'PRESENT_END_SET_TIMER';
 export const DISMISS_END_SET_TIMER = 'DISMISS_END_SET_TIMER';
+export const SET_DEFAULT_METRIC = 'SET_DEFAULT_METRIC';
```

Comment:
Rename SET_DEFAULT_METRIC to SAVE_DEFAULT_METRIC

Reply:
Done